### ISSUE TITLE
Allow students to access the lesson page and submit lesson document

### ISSUE THREAD

**Issue by amellie** [Ali-Jahankah/bugsquashers-edu-app#26]
## Dependency

Depends on #24 

## Story

As a student
I want to see the content of a lesson and download its document
So that I can do the lesson assessments

As a student
I want to be able to upload my lesson document that I have worked on
So that I can submit my work into the lesson

## Acceptance Criteria
- The lesson page should show the module name, lesson name, lesson description, and a link to download a document
- The page should also has the option to upload a document (to create a submission, see #24 on the API to be used)
- If a student has submitted a document, it should not be able to submit again
- The submitted document should be saved into the server, grouped by module ID, e.g. /submissions/1/ (module ID is 1)
- If #25 is already done, please add a link from the student dashboard into the lesson page
